django/core/files/move.py

```diff
@@ -43,13 +43,12 @@ def file_move_safe(
     if _samefile(old_file_name, new_file_name):
         return
 
-    try:
-        if not allow_overwrite and os.access(new_file_name, os.F_OK):
-            raise FileExistsError(
-                "Destination file %s exists and allow_overwrite is False."
-                % new_file_name
-            )
+    if not allow_overwrite and os.access(new_file_name, os.F_OK):
```

Comment:
Can `os.access()` raise `OSError`?

Reply:
If yes, we need to handle this.

Reply:
The [code](https://github.com/python/cpython/blob/main/Modules/posixmodule.c#L3122) seems to not raise any errors, but I could be misreading it. If you think I should catch `OSError`, I'll add it.